src/uk/gov/hmcts/contino/YarnBuilder.groovy

```diff
@@ -158,6 +158,10 @@ class YarnBuilder extends AbstractBuilder {
 
   def prepareCVEReport(String issues, String knownIssues) {
     def jsonSlurper = new JsonSlurper()
+    if (!isYarnV2OrNewer()){
+      WarningCollector.addPipelineWarning("new_suppressions",
```

Comment:
hmm I'm not sure if this is quite right, a nagger should only alert if it detects there's something wrong. which in this case would be to run both and only trigger the nagger if recursive fails but the current version passes.

(which would depend on how much time it adds to the build to add both, if its small then fine, if its not small then an announcement on slack is probably better)

Reply:
Should now work as requested :)